### PR TITLE
convert show/hide details button in custom games to icon

### LINKED ISSUE #1732: Turn show/hide details button in custom games into an icon
This button would take less space and be pretty self-explanatory if it were an arrow icon.
grafik

### PR DESCRIPTION
**Pull request by BlackYps**
The button now takes less space and is more on the right

### CODE REVIEW

**Comment by 1-alex98** [comment]
Screenshot would be nice :D... guess have to look at it for myself

**Comment by BlackYps** [comment]
grafik

**Review by 1-alex98 — DISMISSED** [review]

File: src/main/java/com/faforever/client/game/CustomGamesController.java [hunk 1]
```diff
@@ -280,10 +280,10 @@ public void toggleSidePane() {
 
   private void setSidePane(boolean displayed) {
     if (displayed) {
-      toggleSidePaneButton.setText(i18n.get("view.hideSidePane"));
+      toggleSidePaneButton.setText("");
```
Comment (axel1200) [inline src/main/java/com/faforever/client/game/CustomGamesController.java:283]: Set in fxml

**Comment by BlackYps** [comment]
Changed the icon:

grafik

Fixes #1732

**Comment by 1-alex98** [comment]
I really liked the first one better. The windows arrows are also that way on the other site the vault now has the other arrows so I guess we at least need to make it consistent.

**Comment by Katharsas** [comment]
I think its important to make it completely clear which parts of the GUI are interactive and which are not. If buttons are added, they should imo look like buttons, so i would add some background color change to this. The Table/Tiles view switch could also use that.

**Comment by BlackYps** [comment]
I don't really get what windows arrows you mean, but I agree that the first one was better.
I made it look more like a normal button now, that kind of sits on the edge that it is supposed to collapse:
I also fixed the Table/Tiles toggle buttons, so they look consistent to other buttons.
grafik

**Comment by 1-alex98** [comment]
merging as soon as tests run through... tell me if forget

**Pull request merged**